projects/fusion-ui/components/tabs/v3/tabs/tabs.component.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../../../src/style/scss/v3/vars/vars';
+
+$tabWidth: 150px;
+$tabHeight: 48px;
+$tabPadding: 4px;
+$tabBorderWidth: 1px;
+
+%tabs-flex{
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: var(--fu-tabs-justify-content, flex-start);
+}
+
+:host{
+    @extend %tabs-flex;
+    border-bottom: var(--fu-tab-border-bottom, solid $fu-light-400);
+    border-bottom-width: var(--fu-tab-border-bottom-width, $tabBorderWidth);
+}
+
+// for support additional div for ng-content in native
+:host-context(native-fusion-tabs){
+    & ::ng-deep > div {
```

Comment:
why you need ng-deep here?

Reply:
We need it because we need a set style for the first div in ng-content that is not part of this component, so need `::ng-deep` in case the parent has the tag `native-fusion-tabs`.